API_CHANGELOG.md

```diff
@@ -6,7 +6,12 @@ included in this changelog.
 
 ### Additions
 
-None.
+- `GET /{project_id}/labels/stats`
+  Get statistics on node label usage for the project.
+
+- `POST /{project_id}/skeletons/node_labels`
```

Comment:
Prefer hyphens to underscores in URLs: http://catmaid.readthedocs.io/en/stable/contributing.html#http-api